### PR TITLE
Feat/adjusts multi assets

### PR DESCRIPTION
**Pull request by pedroanastacio**
- Adds BTC to static assets
- Vault balance calculation now considers ETH and BTC
- Workspace balance calculation now considers ETH and BTC
- TVL calculation now considers ETH and BTC
- Endpoint get workspace balance also returns a list with balances per asset
- Endpoint has reserved coins returns an array of assets with reserved amounts
- Changes the API used to obtain coin quotes
- Creates cache on server to store coin quotes

https://app.clickup.com/t/86a3bnrmb

### CODE REVIEW

File: packages/api/src/utils/cronJobs/TVL.ts [hunk 1]
```diff
@@ -45,34 +43,45 @@ const TVLCronJob = cron.schedule('0 0 * * *', async () => {
...
+    );
 
-    // Salva no BD
-    await TotalValueLocked.create({
-      assetId,
-      amount,
-      amountUSD,
-    }).save();
+    // Salva cada item no BD
+    await Promise.all(
```
Comment (guimroque) [inline packages/api/src/utils/cronJobs/TVL.ts:75]: change this loop to for await

File: packages/api/src/utils/balance.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { CoinQuantity, bn } from 'fuels';
+import { assets } from '@src/mocks/assets';
+
+import axios from 'axios';
+
+const getPriceUSD = async (assetSlug: string): Promise<number> => {
+  try {
+    const convert = `${assetSlug}-USD`;
+    const { data } = await axios.get(
+      `https://economia.awesomeapi.com.br/last/${convert}`,
```
Comment (guimroque) [inline packages/api/src/utils/balance.ts:10]: change api consumed to: https://coinmarketcap.com

File: packages/api/src/utils/cronJobs/TVL.ts [hunk 3]
```diff
@@ -1,20 +1,16 @@
...
 // Executa todos os dias a meia-noite
 const TVLCronJob = cron.schedule('0 0 * * *', async () => {
   console.log('[TVL CRON JOB] running');
   try {
-    //Validação para limitar apenas um registro diário por assetId
+    //Validação para limitar apenas um registro diário
     const savedTVL = await TotalValueLocked.createQueryBuilder('tvl')
-      .where('tvl.assetId = :assetId', { assetId })
-      .andWhere('DATE(tvl.created_at) = CURRENT_DATE')
+      .where('DATE(tvl.created_at) = CURRENT_DATE')
```
Comment (guimroque) [inline packages/api/src/utils/cronJobs/TVL.ts:13]: would the validation be just the exact date?
try:
`  .where('tvl.created_at >= CURRENT_DATE AND tvl.created_at < CURRENT_DATE + INTERVAL '1 DAY'')
`

File: packages/api/src/server/storage/quote.ts [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+      }));
+
+      return formattedData;
+    } catch (e) {
+      console.log('[GET_ASSET_PRICE_USD_ERROR]: ', e);
+      return [];
+    }
+  }
+
+  public startDev() {
```
Comment (guimroque) [inline packages/api/src/server/storage/quote.ts:91]: is it better to move these `start` or `startDev` functions to the constructor?

and remove this `initQuotesJob` method in `app.ts`?

File: packages/api/src/utils/cronJobs/TVL.ts (hunk 3 above)
Comment (pedroanastacio) [reply packages/api/src/utils/cronJobs/TVL.ts:13]: Only checking by the current date seems to satisfy the condition of having only one record per day. It checks whether there is any record in the table created on the current date and bases the result on that:
     - If a record for the current day already exists, the cron job should stop.
     - If it doesn't, the cron job should continue and create the necessary records.

**Review by guimroque — APPROVED** [review]

**Pull request merged**